lib/utils/style.dart

```diff
@@ -103,19 +103,17 @@ class Style {
 
   static Widget button(text, onPressed,
       {color, textColor, borderColor, elevation}) {
-    return RaisedButton(
+    return ElevatedButton(
       onPressed: onPressed,
-      elevation: 0,
-      highlightElevation: 0,
-      hoverElevation: 0,
-      highlightColor: color ?? colorPrimary.withOpacity(0.5),
-      padding: EdgeInsets.symmetric(vertical: 14),
-      color: color ?? colorPrimary,
-      disabledColor: color != null
-          ? color.withOpacity(0.3)
-          : colorPrimary.withOpacity(0.3),
-      shape: RoundedRectangleBorder(
-        borderRadius: BorderRadius.circular(3),
+      style: ElevatedButton.styleFrom(
+        elevation: 0,
+        tapTargetSize: MaterialTapTargetSize.shrinkWrap,
+        padding: EdgeInsets.symmetric(vertical: 14),
+        backgroundColor: color ?? colorPrimary,
+        foregroundColor: textColor ?? Colors.white,
```

Comment:
I am getting the following error on multiple places of this file for both `backgroundColor` and `foregroundColor`. It's most likely due to a different version of flutter/dart or dependencies
```
The named parameter 'backgroundColor' isn't defined.
Try correcting the name to an existing named parameter's name, or defining a named parameter with the name 'backgroundColor'.dart[undefined_named_parameter](https://dart.dev/diagnostics/undefined_named_parameter)
```

Reply:
No problem @DigitalSeneca. I am now reviewing the feedback

> Seems like we're having some conflict here with firebase_auth, so I can't compile on Android after the dependencies upgrade. Are you using a different version of flutter/dart? 

Yes it looks like I am using a more recent version. See below.

flutter --version
Flutter 3.13.7 • channel stable • https://github.com/flutter/flutter.git
Framework • revision 2f708eb839 (6 months ago) • 2023-10-09 09:58:08 -0500
Engine • revision a794cf2681
Tools • Dart 3.1.3 • DevTools 2.25.0